.changeset/afraid-frogs-drop.md

```diff
@@ -0,0 +1,5 @@
+---
+'@codeimage/app': major
```

Comment:
Here the changeset should be “minor”. That’s because following semantic versioning we have

- fix: patch
- feat: minor
- major: feat with breaking changes (but CodeImage didn’t react the 1.0 yet so each feat should be minor even if with breaking changes

Reply:
sorry, my bad! I'm done proceeding without reading properly..